packages/compass-schema/src/modules/schema-analysis.ts

```diff
@@ -81,8 +84,11 @@ export const analyzeSchema = async (
     log.error(mongoLogId(1001000091), 'Schema', 'Schema analysis failed', {
       ns,
       error: err.message,
+      aborted: abortSignal.aborted,
     });
-    if (dataService.isCancelError(err)) {
+
+    if (abortSignal.aborted) {
+      // The operation was aborted, so we don't throw an error.
```

Comment:
Do we ever give abort() a `reason`? Should it be logged/captured here either way?

Reply:
Added to the log! We typically do pass a reason, not sure if it is all that actionable or useful, but maybe down the line it will be.